src/posts/hello-day-8.md

```diff
@@ -150,13 +152,15 @@ Well... the Sass sitemaps built properly, but none of the Github Actions env stu
 
 What if I set the `env` at the level of job? I think this means I could prob use `GITHUB_SHA`, but I want to see what works.
 
+{% raw %}
 ```yaml
 jobs:
   deploy:
     runs-on: ubuntu-latest
     env:
       MY_GITHUB_RUN_ID: ${{ github.run_id }}
 ```
+{% endraw %}
```

Comment:
Same comment as above, where Nunjucks tries to evaluate `{{ github.run_id }}` and throws an error due to `throwOnUndefined: true`.